adapters/pulsepoint/pulsepoint.go

```diff
@@ -1,44 +1,196 @@
 package pulsepoint
 
 import (
-	"bytes"
-	"context"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
 	"net/http"
 	"strconv"
-	"strings"
 
+	"bytes"
+	"context"
 	"github.com/mxmCherry/openrtb"
 	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
 	"github.com/prebid/prebid-server/errortypes"
 	"github.com/prebid/prebid-server/openrtb_ext"
 	"github.com/prebid/prebid-server/pbs"
 	"golang.org/x/net/context/ctxhttp"
+	"io/ioutil"
+	"strings"
 )
 
 type PulsePointAdapter struct {
 	http *adapters.HTTPAdapter
 	URI  string
 }
 
+// Builds an instance of PulsePointAdapter
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter) (adapters.Bidder, error) {
+	bidder := &PulsePointAdapter{
+		URI: config.Endpoint,
+	}
+	return bidder, nil
+}
```

Comment:
Please add a test for the Builder func. The template you can copy/paste is here: https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#adapter-code-tests

We generally recommend the use of our newer json file test framework instead of using test functions. Please consider using the framework rather than adding new unit tests such as `TestMakeBids`. If you feel strongly against doing this, please at least use the `Builder` instead of newing up a `PulsePointAdapter` within your unit tests.

Reply:
addressed